docs/topics/files.txt

```diff
@@ -202,3 +202,31 @@ For example, the following code will store uploaded files under
 :doc:`Custom storage systems </howto/custom-file-storage>` work the same way:
 you can pass them in as the ``storage`` argument to a
 :class:`~django.db.models.FileField`.
+
+Using a callable
+----------------
+
```

Comment:
Could put in a `.. versionchanged:` here?

Reply:
It's a new section, so I would add a `versionadded` at the beginning:
```
Using a callable
----------------

.. versionadded:: 3.1

You can use ...
```